### ISSUE TITLE
Glossy lava monkey glitch?

### ISSUE THREAD

**Issue by GermanTheDev** [NachoEngine/GorillaTag-Player-IK-Blender-Rig#4]
When I open the .blend file, it gives me a lava monkey with a name tag with no outline, as well as a monkey with ONLY the lava texture (meaning no face or body fur).

**Comment by Slothgam3r** [comment]
did you find i fix. i have the same problem

**Comment by GermanTheDev** [comment]
yes i found the fix thanks to smokevr, i was told to download the steam version of blender instead of downloading it from the website. hope it helps!

**Comment by TimberTheSilly** [comment]
What do I do if I can't download the steam version of blender and I have this issue

**Comment by EatSleepCSRepeat** [comment]
Maybe try redownloading the blend file
